README.md

```diff
@@ -1365,10 +1365,14 @@ For example, here's how a response handler using [Ono](https://github.com/mattt/
 extension DataRequest {
     static func xmlResponseSerializer() -> DataResponseSerializer<ONOXMLDocument> {
         return DataResponseSerializer { request, response, data, error in
+            // Pass through any underlying URLSession error to the .network case.
             guard error == nil else { return .failure(BackendError.network(error: error!)) }
 
-            guard let validData = data else {
-                return .failure(BackendError.dataSerialization(reason: "Input data was nil."))
+            // Use Alamofire's existing data serializer to extract the data, passing the error as nil, as it has
+            // alreaady been handled.
+            let result = Request.serializeResponseData(response: response, data: data, error: nil)
```

Comment:
Added in [0a05e2b](https://github.com/Alamofire/Alamofire/commit/880dc772a12d1ec490eb8aee029115ec8).